src/components/containers/show.js

```diff
@@ -30,14 +30,14 @@ export const Mobile = ({ children, ...props }) => (
 export const Eu = ({ children }) => {
     const { is_eu_country } = React.useContext(LocationContext)
 
-    if (is_eu_country) return <>{children}</>
+    if (is_eu_country) return <div>{children}</div>
```

Comment:
The `div` container should not be handled here imo, in case you want to add styling or use a different element.

Reply:
actually, using react fragment makes the children for EU failed to render, @suthesh finds out that its fixed by adding div in show eu component

Reply:
but yes, it should be handled in different pr